packages/devtools_app_shared/lib/src/service/eval_on_dart_library.dart

```diff
@@ -45,7 +45,7 @@ class EvalOnDartLibrary extends DisposableController
     _libraryRef = Completer<LibraryRef>();
 
     // For evals in tests, we will pass the isolateId into the constructor.
-    isolate ??= serviceManager.isolateManager.selectedIsolate;
+    isolate ??= serviceManager.isolateManager.mainIsolate;
```

Comment:
I don't think we want to do this for all EvalOnDartLibrary instances. Just for the one that the inspector creates

Reply:
ah yes, sorry went too fast and only looked at the inits 🖖